src/main/java/es/unizar/smartcampuz/application/controller/UserController.java

```diff
@@ -206,14 +202,8 @@ else if(!(user.getPassword().equals(pass))){
     /*
      * Checks if the username and password fields are null or empty.
      */
-    private boolean verifyFields(String username, String pass){
-        if((username==null) || (username.trim().equals(""))){
-            return false;
-        }
-        if((pass==null) || (pass.trim().equals(""))){
-            return false;
-        }
-        return true;
+    private boolean notBlank(String field){
```

Comment:
I think this should be the other way. Right now "notBlank" is returning "true" if the field is null or blank, and it should return "false" 'cause it's false that the field is not blank. Maybe it's not a good name for a method.

Besides, please review line 83 where you are calling this method, 'cause it's use is not right because of what it's returning.